### PR TITLE
Update to 0.7.2

### PR DESCRIPTION
**Pull request by dyamon**

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -2,15 +2,15 @@ pkgdesc="ROS - This package contains generic definitions of geometric shapes and
...
 arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
-pkgrel=4
+pkgrel=1
 license=('BSD')
 
 ros_makedepends=(
     ros-noetic-resource-retriever
     ros-noetic-eigen-stl-containers
-    ros-noetic-catkin
+    'ros-noetic-catkin>=0.5.68'
```
Comment (acxz) [inline PKGBUILD]: Typically versioned deps are heavily frowned upon. There is only one package that provides ros-noetic-catkin which we control so there is no need to specify a lowerbound for this.

**Review by acxz — CHANGES_REQUESTED** [review]
Just one nitpick. But otherwise looks good. Thx so much, really helps out!

File: PKGBUILD (hunk 1 above)
Comment (dyamon) [reply PKGBUILD]: Oh ok, good to know.

**Comment by acxz** [comment]
LGTM

**Pull request merged**